core/cmd/app.go

```diff
@@ -270,6 +270,12 @@ func NewApp(s *Shell) *cli.App {
 					FileMaxBackups: int(s.Config.Log().File().MaxBackups()),
 					SentryEnabled:  s.Config.Sentry().DSN() != "",
 				}
+
+				// early initialization of the logger with Otel core
+				if cfg.Telemetry().LogStreamingEnabled() {
+					lggrCfg.AdditionalCores = append(lggrCfg.AdditionalCores, logger.NewOtelCore())
+				}
```

Comment:
Do we need such a general solution? Or could we keep it simple:
```suggestion
				// early initialization of the logger with Otel core
				if cfg.Telemetry().LogStreamingEnabled() {
					lggrCfg.TelemetryStreamingEnabled = true
				}
```
Then `NewOtelCore` doesn't need to be exported.

Reply:
Or pass the provider here, so that logger.NewOtelCore() doesn't have to call global funcs
```suggestion
				// early initialization of the logger with Otel core
				if cfg.Telemetry().LogStreamingEnabled() {
					// note: until beholder.SetGlobalOtelProviders() is not called
					// the core uses no-op provider and logger
					lggrCfg.TelemetryLoggerProvider = otellogglobal.GetLoggerProvider()
				}
```